src/app/containers/StakePage/components/HistoryEventsTable.tsx

```diff
@@ -222,35 +153,42 @@ interface HisoryAsset {
   index: number;
 }
 
-const HisoryTableAsset: React.FC<HisoryAsset> = ({ item, index }) => {
+const HisoryTableAsset: React.FC<HisoryAsset> = ({ item }) => {
   const { t } = useTranslation();
   const SOV = AssetsDictionary.get(Asset.SOV);
   const dollars = useCachedAssetPrice(Asset.SOV, Asset.USDT);
   const dollarValue = useMemo(() => {
-    if (item.returnVal.amount === undefined) return '';
-    return bignumber(item.returnVal.amount)
+    if (item.amount === undefined || null) return '';
```

Comment:
Rewrite it to `if(!item.amount)` and don't write one-line `if`.

Reply:
+